app/assets/javascripts/admin/enterprises/controllers/side_menu_controller.js.coffee

```diff
@@ -44,3 +45,12 @@ angular.module("admin.enterprises")
 
     $scope.enterpriseIsShop = ->
       $scope.Enterprise.sells != "none"
+
+  window.addEventListener 'load', ->
+    element = document.getElementById('enterprise_permissions')
+    if !element
+      return
+    element.addEventListener 'click', ->
+      window.location.replace('/admin/enterprise_relationships')
+      return
+    return
```

Comment:
This is a bit ugly. Difficult to solve though because the SideMenu was designed as a menu for tabs, not for links to other pages. We could extend that though. We could add an href attribute and handle that in the service:
https://github.com/openfoodfoundation/openfoodnetwork/blob/729e9eda668fbaface4bb0101cb48d80d0b37b3b/app/assets/javascripts/admin/side_menu/services/side_menu.js.coffee#L22-L27

Reply:
@mkllnk Do you think it's better to do an optional param?

Reply:
Yes, I do. Otherwise it may be hard to find out why this one menu item is behaving differently. Having functionality spread around the code base is very confusing for new developers.